tests/gocase/unit/type/list/list_test.go

```diff
@@ -1000,4 +1000,100 @@ func TestList(t *testing.T) {
 			require.Empty(t, rdb.LPosCount(ctx, "mylist", "b", 10, redis.LPosArgs{Rank: 5}).Val())
 		})
 	}
+
+	for _, direction := range []string{"LEFT", "RIGHT"} {
+		key1 := "lmpop-list1"
+		key2 := "lmpop-list2"
+		rdb.Del(ctx, key1, key2)
+		require.EqualValues(t, 5, rdb.LPush(ctx, key1, "one", "two", "three", "four", "five").Val())
+		require.EqualValues(t, 5, rdb.LPush(ctx, key2, "ONE", "TWO", "THREE", "FOUR", "FIVE").Val())
+		t.Run(fmt.Sprintf("LMPOP-test-oneKey-countSingle-%s", direction), func(t *testing.T) {
+			result := rdb.LMPop(ctx, direction, 1, key1)
+			resultKey, resultVal := result.Val()
+			require.NoError(t, result.Err())
+			require.EqualValues(t, key1, resultKey)
+			if direction == "LEFT" {
+				require.Equal(t, []string{"five"}, resultVal)
+			} else {
+				require.Equal(t, []string{"one"}, resultVal)
+			}
+		})
+		t.Run(fmt.Sprintf("LMPOP-test-oneKey-countMulti-%s", direction), func(t *testing.T) {
```

Comment:
please try to remove the `-` in the test name, also, please leave a newline between the each test
```suggestion

		t.Run(fmt.Sprintf("LMPOP test-oneKey-countMulti-%s", direction), func(t *testing.T) {
```

Reply:
Refactored to meet your suggestions.